binding.js

```diff
@@ -95,19 +95,22 @@ module.exports.download_vips = function () {
           } catch (err) {}
         });
       });
-      const gotOpt = {
+      const url = distBaseUrl + tarFilename;
+      const simpleGetOpt = {
+        url,
         agent: caw(null, {
           protocol: 'https'
         })
       };
-      const url = distBaseUrl + tarFilename;
-      got.stream(url, gotOpt).on('response', function (response) {
+      simpleGet.stream(simpleGetOpt, function (err, response) {
```

Comment:
Is this correct? Perhaps `simpleGet(simpleGetOpt, ...`?

Reply:
Oops, yeah. You're right. Fixing.